src/main/java/com/google/devtools/build/lib/analysis/BUILD

```diff
@@ -696,6 +696,7 @@ java_library(
     name = "configured_target",
     srcs = ["ConfiguredTarget.java"],
     deps = [
+        ":config/config_matching_provider",
```

Comment:
This is totally orthogonal to your PR and just something I've never understood - why have a slash in a target name as opposed to just putting it under a directory called config?

Reply:
Because [analysis_cluster](https://cs.opensource.google/bazel/bazel/+/master:src/main/java/com/google/devtools/build/lib/analysis/BUILD;l=164) contains files from inside the config subdirectory, config can't be its own package with its own BUILD file.

Ideally we'd further break up analysis_cluster and fix this, but it's a hard problem and I couldn't crack it last time I tried.